### ISSUE TITLE
Set default color map and default first displayed scalar field  for lasfile

### ISSUE THREAD

**Issue by gvoysey** [CloudCompare/CloudCompare#2042]
**Describe the feature you would like**
When opening multiple lasfiles at once (say, 10-15 tiles of a scene), I believe I currently have to change the color map and displayed scalar field for each lasfile manually in turn, in the GUI. 

I would like to be able to specify a way to bulk-open all these lasfiles and for each of them, set the default color map to a particular custom color map I have already defined, set the number of steps to 999 steps, and set the displayed scalar field to a particular SF that I know will be present in each.

**Comment by dgirardeau** [comment]
Yes, I believe that the ability to change entity properties in a bulk is already in the TODO list (and for a long time).

There's a very partial workaround which is to use the command line to load all the tiles, change the current scalar field and the current color scale, and save everything as one big BIN file 😅. But you would have to write a script to do that on a dynamic list of files selected in the explorer for instance. And you won't be able to change the number of steps. Plus, it will probably create a huge file...

**Comment by tmontaigu** [comment]
This is something that should be somewhat easy to do using  python script

**Comment by gvoysey** [comment]
where's the python API documented?  (i'm very familiar with python development, just don't know where to wire things in for cloudcompare)

**Comment by tmontaigu** [comment]
The project is here https://github.com/tmontaigu/CloudCompare-PythonRuntime
and the docs here https://tmontaigu.github.io/CloudCompare-PythonRuntime/index.html
